spec/models/requests/requestable_spec.rb

```diff
@@ -384,15 +384,15 @@
 
   context 'A requestable serial item that has volume and item data in its openurl' do
     let(:request) { FactoryGirl.build(:aeon_rbsc_enumerated, patron: patron) }
-    let(:requestable_holding) { request.requestable.select { |r| r.holding['6720550'] } }
+    let(:requestable_holding) { request.requestable.select { |r| r.holding['22127290370006421'] } }
```

Comment:
@kevinreiss I'm curious on this one.  Did you need to change the holding item?  That leads me to believe that some items have the old keys and some have the new keys.  Would it be better to use either key